8. Appels réseaux/Content/part1.md

```diff
@@ -231,6 +233,8 @@ private static let quoteUrl = URL(string: "https://api.forismatic.com/api/1.0/")
 
 > **:information_source:** Je vous suggère de bien indiquer en haut de vos classes vos URL pour qu'elles soient bien visibles et donc modifiables facilement si besoin.
 
+> **:information_source:** Sur de plus gros projets où vous utilisez votre propre API, je vous conseille de séparer la racine de l'url de la partie variable. Si vous avez de nombreux appels réseaux et que vous changez de version d'API, cela vous évitera de changer *toutes* vos URL.
```

Comment:
Something like declaring a baseURL, and also a version that you can always override if need be.